instrumentation/google.golang.org/grpc/otelgrpc/example/api/hello-service.pb.go

```diff
@@ -5,9 +5,11 @@
 Package api is a generated protocol buffer package.
 
 It is generated from these files:
+
```

Comment:
Why are those new lines here? They seem unrelated to this PR.

Reply:
Hmm. Not sure. Let me back those out

Reply:
Fixed in f54fc6c7805342222601f1edfc27d94d642fc9a0